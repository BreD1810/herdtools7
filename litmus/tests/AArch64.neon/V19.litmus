AArch64 V19
(* Tests MOV vector *)
{
0:X0=x;0:X1=y; 0:X2=z;
}

 P0                     ;
 MOVI V0.8B,#1          ;
 MOV V1.8B,V0.8B        ;
 ST1 {V1.B}[0], [X0]    ;
 ST1 {V1.B}[7], [X1]    ;
 ST1 {V1.B}[8], [X2]    ;
 forall (x=1 /\ y=1 /\ z=0)
