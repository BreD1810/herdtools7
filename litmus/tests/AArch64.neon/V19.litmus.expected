Test V19 Required
Histogram (1 states)
1000000:>x=1; y=1; z=0;
Ok

Witnesses
Positive: 1000000, Negative: 0
Condition forall (x=1 /\ y=1 /\ z=0) is validated
Hash=5e23c64de2f65cfa8defa6d9351a31e3
Observation V19 Always 1000000 0
